pkgs/applications/networking/browsers/firefox/common.nix

```
@@ -83,6 +83,7 @@
 , gssSupport ? true, libkrb5
 , jemallocSupport ? true, jemalloc
 , ltoSupport ? (stdenv.isLinux && stdenv.is64bit), overrideCC, buildPackages
+, pgoSupport ? (stdenv.isLinux && stdenv.isx86_64), xvfb-run
```

```suggestion
, pgoSupport ? (stdenv.isLinux && stdenv.isx86_64 && hostPlatform == buildPlatform), xvfb-run
```

PGO should be disabled when cross-compiling.